zstacklib/zstacklib/utils/lv_metadata.py

```
@@ -0,0 +1,948 @@
+                     vg_uuid, len(metadata_lvs), len(entries))
+        return entries
+
+    def _do_cleanup(self, metadataPath):
+        lvm = self._lvm
+
+        try:
+            if lvm.lv_exists(metadataPath):
+                delete_metadata_lv(metadataPath, lvm.delete_lv)
+            else:
+                logger.debug("metadata LV %s does not exist, skip cleanup", metadataPath)
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)